src/templates/cards.tag

```diff
@@ -0,0 +1,247 @@
+<offer-card>
+    <div class="card_table">
+        <div class="logo" style={parent.cssStyleRules(undefined, item.primary_color)} id={item.id}>
+            <img src={item.logo}></img>
+        </div>
+        <h1 style={parent.cssStyleRules(item.secondary_color)} class="name" id={item.id}>{item.name}</h1>
+        <raw class="usage" content="{item.description}"></raw>
+        <a class ="details" style={parent.cssStyleRules(item.secondary_color)} href={parent.formatDetailsUrl(item.repositoryId, item.id)}>More Details</a>
+        <a href="#" style={parent.cssStyleRules('#fff', item.primary_color)} class="button" id={item.id}> Buy for {parent.formatMoney(item.price.value, item.price.unit)}</a>
+        <div class="footer-logo">
+            <p>Powered by </p> <img src="../src/templates/logos/opp-logo.png"></img>
+        </div>
+    </div>
+</offer-card>
+
+<organisation-card>
+    <div class="card_table">
+        <div class="logo" style={parent.cssStyleRules(undefined, item.primary_color)} id={item.id}>
+            <img src={item.logo}></img>
+        </div>
+        <h1 style={parent.cssStyleRules(item.secondary_color)} class="name" id={item.id}>{item.name}</h1>
+        <raw class="usage" content="{item.description}"></raw>
+        <a if={item.link} href={item.link} style={parent.cssStyleRules('#fff', item.primary_color)}  class="button" id={item.id}> Go to Site</a>
+        <div class="footer-logo">
+            <p>Powered by </p> <img src="../src/templates/logos/opp-logo.png"></img>
+        </div>
+    </div>
+</organisation-card>
+
+<cards>
+    <div class="all">
+        <div class="swiper-container swiper" id="container">
+            <div class="swiper-wrapper" id="wrapper">
+                <div each={item in this.items} class="swiper-slide">
+                    <offer-card if={parent.type=='offer'} item={item}/>
+                    <organisation-card if={parent.type=='licensor'} item={item}/>
+                    <organisation-card if={parent.type=='link'} item={item}/>
+                </div>
+            </div>
+            <div class="swiper-pagination swiper-pagination-white" id="pagination"></div>
+            <div class="swiper-button-next swiper-button-white"></div>
+            <div class="swiper-button-prev swiper-button-white"></div>
+        </div>
+    </div>
+
+<script>
```

Comment:
should these script blocks be indented? I thought you had forgotten the closing `<\cards>`